agent-tee-phala/scripts/test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This script seems irrelevant, i will propose later what pieces can be salvaged but putting this as a mental note